### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by vespasianvs**

### CODE REVIEW

File: config/test.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  // enable verbose logging to std::out (contains DIMSE output)
+  "verboseLogging": true,
+
+  // only disable when using wadouri instead of wadors
+  "fullMeta": true,
+
+  // disabled when empty, otherwise use connection url e.g. ws://localhost:6001
+  "websocketUrl": "http://127.0.0.1:6001",
+
+  // this is the secret your dicomweb-websocket-bride needs to configure too, make sure to keep it secret, and of course change it ,-) hint: use https://hashgenerator.de/
```
Comment (nickhingston) [inline config/test.json]: typo

File: src/dimse/fetchGet.ts [hunk 2]
```diff
@@ -50,13 +50,16 @@ export async function fetchGet(studyUid: string, seriesUid: string, imageUid: st
         if (result && result.length > 0) {
           try {
             const json = JSON.parse(result);
-            if (json.code === 0 || json.code === 2) {
+            if (json.code === 0) {
               logger.info(`fetch finished: ${uidPath}`);
               resolve(result);
             }
-            else {
+            else if(json.code === 1) {
```
Comment (nickhingston) [inline src/dimse/fetchGet.ts]: can we have space after if

Comment (nickhingston) [reply src/dimse/fetchGet.ts]: lint?

Comment (vespasianvs) [reply src/dimse/fetchGet.ts]: Not part of eslint:recommended

**Review by nickhingston — APPROVED** [review]

**Pull request merged**